system.go

```diff
@@ -1,32 +1,28 @@
 package akara
 
-import "time"
-
 // System describes the bare minimum of what is considered a system
 type System interface {
+	Init(*World)
+	IsInitialized() bool
 	Active() bool
 	SetActive(bool)
 	Destroy()
-}
-
-type baseSystem interface {
-	System
-	bind(*World)
-}
-
-// SystemInitializer is a system with an Init method
-type SystemInitializer interface {
-	System
-	Init(*World)
-	IsInitialized() bool
-}
```

Comment:
This interface was broken out to allow for systems that don't have any initialization logic, basically just to reduce `noop` methods an end user might need to make when creating a new system.

Reply:
For the time being, I ended up reverting back to something like this SystemInitializer thing